package.json

```diff
@@ -11,12 +11,15 @@
   },
   "dependencies": {
     "@next/font": "^13.3.0",
+    "dropdown": "^0.2.0",
```

Comment:
I don't think you are using this package

```suggestion
```

Reply:
with it I wanted to build something that I then solved differently

Reply:
top thx

Reply:
Thanks for all the great additions. Is taken over!